google-apis-generator/lib/google/apis/generator/templates/repo-metadata.json.tmpl

```diff
@@ -0,0 +1,7 @@
+{
+    "api_id": "<%= api.id %>",
+    "name_pretty": "<%= api.title %>",
+    "distribution_name": "<%= api.gem_name %> API",
```

Comment:
The "API" should be on the name_pretty (i.e. title) field, not the distribution_name (which is the gem name).

Reply:
Nice catch! I double checked, the api.title has the word "API" included in it already. I'll remove the "API"